From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: George T Kramer <george.t.kramer@intel.com>
Date: Fri, 25 Jan 2019 16:42:48 -0800
Subject: [PATCH] Prepare systemd service

---
 tools/stunnel.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/stunnel.service.in b/tools/stunnel.service.in
index 11f848d..1b306fe 100644
--- a/tools/stunnel.service.in
+++ b/tools/stunnel.service.in
@@ -1,6 +1,6 @@
 [Unit]
 Description=TLS tunnel for network daemons
-After=syslog.target network-online.target
+After=network-online.target
 
 [Service]
 ExecStart=@bindir@/stunnel
